docs/user/registering-microshift-devices-acm.md

```diff
@@ -0,0 +1,157 @@
+# Auto-Registering Devices with MicroShift into ACM
+
+## Configuring RHACM Agent Registration
```

Comment:
The user documentation follows a specific pattern that we should also adopt for this section (but we can update that in a follow-up PR):

1. First give an overview of what the section is about ("If you have a fleet of devices running MicroShift and workloads on MicroShift to be managed through ACM, you can set up the fleet so that MicroShift instances get automatically registered into ACM.") and a general explanation of concepts+processes.
2. Implementing this using the UI.
3. Implementing this using the CLI.

Reply:
Let's do it in a follow up PR, because the UI is not ready for hooks yet.